[2023-09-04T06:03:08.178+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-09-04T06:03:08.193+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-09-04T06:03:08.194+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:03:08.214+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-05-22 00:00:00+00:00
[2023-09-04T06:03:08.225+0000] {standard_task_runner.py:57} INFO - Started process 5699 to run task
[2023-09-04T06:03:08.229+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-05-22T00:00:00+00:00', '--job-id', '599', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmppke0tf8w']
[2023-09-04T06:03:08.232+0000] {standard_task_runner.py:85} INFO - Job 599: Subtask insert_into_table
[2023-09-04T06:03:08.308+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:03:08.439+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-22T00:00:00+00:00'
[2023-09-04T06:03:08.441+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-05-22', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:03:08.450+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:08.460+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:08.473+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-05-22', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:03:08.478+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:03:08.497+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220522T000000, start_date=20230904T060308, end_date=20230904T060308
[2023-09-04T06:03:08.562+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:03:08.579+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:58.290+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-09-04T06:05:58.312+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-09-04T06:05:58.312+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:58.341+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-05-22 00:00:00+00:00
[2023-09-04T06:05:58.353+0000] {standard_task_runner.py:57} INFO - Started process 7004 to run task
[2023-09-04T06:05:58.358+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-05-22T00:00:00+00:00', '--job-id', '1147', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmphs_vneex']
[2023-09-04T06:05:58.363+0000] {standard_task_runner.py:85} INFO - Job 1147: Subtask insert_into_table
[2023-09-04T06:05:58.487+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-22T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:58.693+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-22T00:00:00+00:00'
[2023-09-04T06:05:58.695+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-05-22', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:58.713+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:58.733+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:58.757+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-05-22', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:58.766+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:58.807+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220522T000000, start_date=20230904T060558, end_date=20230904T060558
[2023-09-04T06:05:58.892+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:58.925+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
